### PR TITLE
Added constructor as part of Solidity ABI

### PR DESCRIPTION
**Pull request by LianaHus**
issue #1960

### CODE REVIEW

File: libsolidity/InterfaceHandler.cpp [hunk 1]
```diff
@@ -107,17 +107,28 @@ std::unique_ptr<std::string> InterfaceHandler::getABIInterface(ContractDefinitio
 unique_ptr<string> InterfaceHandler::getABISolidityInterface(ContractDefinition const& _contractDef)
 {
 	string ret = "contract " + _contractDef.getName() + "{";
+
+	auto populateParameters = [](vector<string> const& _paramNames,
+			vector<string> const& _paramTypes)
```
Comment (chriseth) [inline libsolidity/InterfaceHandler.cpp]: Might this fit one line now? If not, please correct indentation.

**Comment by chriseth** [comment]
Hm, just noticed that this code never generated the "returns (...)" part. We might add that too.

**Comment by LianaHus** [comment]
yeah we never have "returns....".
do you want me to do it in this PR?

**Comment by chriseth** [comment]
Does not need to be in this PR.

**Comment by chriseth** [comment]
Failure in Linux C++ is probably due to out of memory on the build server.

**Pull request merged**